src/views/Simulation/Recapitulatif.vue

```diff
@@ -135,12 +136,13 @@ export default {
           ENTITIES_PROPERTIES[step.entity].loadEntity &&
           ENTITIES_PROPERTIES[step.entity].loadEntity({
             ...this,
+            $store: this.$store,
```

Comment:
Pourquoi est-ce nécessaire ? `...this` n'inclut pas le `$store` ?

Reply:
En fait ça ne pose plus de problème de juste spread `this` 🤔